src/html/tags/HtmlTagDel.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
We don't really need a specific `HtmlTagDel` component.
We want a component only if the behavior will be quite different than the rest.
If it is just about styling, we can do it in the 'cascadingStyles'.